packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -31,6 +31,12 @@
   import { getConversionPaymentValues } from "../../utils/getConversionPaymentValues";
   import { getEthersSigner } from "../../utils";
 
+  interface BuyerSelerInfo {
```

Comment:
Could you fix the typo here? I think you can also do something like this.

```
interface EntityInfo {
    value: string;
    isCompany?: boolean;
    isEmail?: boolean;
}

interface BuyerInfo extends EntityInfo;

interface SellerInfo extends EntityInfo;

```

Reply:
It should be like this actually:

```
  interface BuyerInfo extends EntityInfo {}

  interface SellerInfo extends EntityInfo {}
```

But yeah I added it